2.4/examples/replica/mongodb-clustered.json

```diff
@@ -94,7 +94,13 @@
                 "command": [
                   "run-mongod", "initiate"
                 ],
-                "containerName": "mongodb"
+                "containerName": "mongodb",
+                "env": [
+                  {
+                    "name": "MONGODB_NODES_TO_WAIT",
```

Comment:
@mfojtik yes, it's only for the replica set initialization.

> if former, I would name this INITIAL_REPLICA_COUNT or something similar.

Just `INITIAL_REPLICA_COUNT` or `MONGODB_INITIAL_REPLICA_COUNT`?